CppCoreGuidelines.md

```diff
@@ -1790,7 +1790,7 @@ If you can't use exceptions (e.g., because your code is full of old-style raw-po
     // ... use val ...
 
 This style unfortunately leads to uninitialized variables.
-A facility [structured bindings](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0144r1.pdf) to deal with that will become available in C++17.
+A facility "structured bindings" to deal with that is available since C++17.
```

Comment:
The grammar here could be improve, how about:

> Since C++17 the "structured bindings" feature can be used to initialize variables directly from the return value:

Reply:
Done.